### PR TITLE
Showing Login Error message

### PR DESCRIPTION
**Pull request by KintonHiryuu**
Showing the error message when login to WFM under the password box instead of only the status
https://i.imgur.com/siV9zpr.png

### CODE REVIEW

File: WFInfo/Login.xaml.cs [hunk 1]
```diff
@@ -1,124 +1,159 @@
...
+                    {
+                        StatusMessage = "Too many requests";
+                        StatusSeverity = 1; //default to too many requests
+                    }
+                    Main.SignOut();
+                    Main.StatusUpdate(StatusMessage, StatusSeverity); //Changing WFinfo status
+
+                    switch (StatusSeverity)
+                    { // copy/paste from Main.cs (statusChange())
+                        case 0: //default, no problem
```
Comment (dapal-003) [inline WFInfo/Login.xaml.cs:99]: Default displays no message so this case is not needed

File: WFInfo/Login.xaml.cs [hunk 2]
```diff
@@ -1,124 +1,159 @@
...
+                try
+                {
+                    await Main.dataBase.GetUserLogin(Email.Text, Password.Password);
+                }
+                catch (Exception ex)
+                {
+                    Main.dataBase.JWT = null;
+                    Settings.Save();
+                    Main.AddLog("Couldn't login: " + ex);
+                    var StatusMessage = "Status"; //StatusMessage = text to display on StatusUpdate() AND the error box under login
```
Comment (dapal-003) [inline WFInfo/Login.xaml.cs:65]: Strange initialization, there's always a value written to this so you can just make it a String.empty

File: WFInfo/Login.xaml.cs [hunk 3]
```diff
@@ -1,124 +1,159 @@
...
+                            break;
+                        default: //Uncaught, big problem
+                            Error.Foreground = Brushes.Yellow;
+                            break;
+                    }
+                    Error.Text = StatusMessage; //Displaying the error under the text fields
+                    if(Error.Visibility != Visibility.Visible)
+                    {
+                        Height += 20;
+                        BackupButton.VerticalAlignment = VerticalAlignment.Center;
```
Comment (dapal-003) [inline WFInfo/Login.xaml.cs:116]: No need to change the vertical alignment if you align the item at the bottom. It'll dynamically stay at the bottom

Comment (KintonHiryuu) [reply WFInfo/Login.xaml.cs:116]: i tried it, it seems like it didnt, but i may used some wrong functions

File: WFInfo/Login.xaml.cs (hunk 2 above)
Comment (KintonHiryuu) [reply WFInfo/Login.xaml.cs:65]: test forget, was initially only var StatusMessage;

File: WFInfo/Login.xaml.cs [hunk 4]
```diff
@@ -1,124 +1,159 @@
...
+                    var StatusSeverity = 0; //StatusSeverity = Severity for StatusUpdate()
+                    if (ex.Message.Contains("email"))
+                    {
+                        if (ex.Message.Contains("app.form.invalid"))
+                        {
+                            StatusMessage = "Invalid email form";
+                            StatusSeverity = 2;
+
+                        }
+                        else
```
Comment (dapal-003) [inline WFInfo/Login.xaml.cs:75]: if else is not properly structured, results that the "else" state always gets executed and thus "invalid email form" never gets properly used.

Comment (KintonHiryuu) [reply WFInfo/Login.xaml.cs:75]: how is it wrong in that case ? cause double enter ?

Comment (dapal-003) [reply WFInfo/Login.xaml.cs:75]: The "else" statement without any curly brackets only executes the next line of code. The following code is no longer seen as the "else" statement and will always be ran. It was the severity that didn't get changed not the message

**Pull request merged**